modules/network-config-cr.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-network-operator.adoc
+[id="network-default-cr-{context}"]
+= Cluster network configuration custom resource
+
+The basic configuration for the `Network Operator` is set in the `Network`
+configuration in a `config.openshift.io` Custom Resource (CR) object. All fields
```

Comment:
Please check all the instances of `config.openshift.io`. Based on one of of @squeed's comments, I'm not sure that you're using this right.